src/csvcubed/utils/sparql_handler/sparqlmanager.py

```diff
@@ -51,6 +53,18 @@
 _logger = logging.getLogger(__name__)
 
 
+class CSVWShape(Enum):
```

Comment:
Can we call this `CubeShape` for consistency? A code list will never be in either of these shapes.